gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/task/AbstractJKubeTask.java

```diff
@@ -85,6 +87,10 @@ protected String getLogPrefix() {
     return configs;
   }
 
+  public KitLogger createLogger(String prefix) {
+    return new GradleLogger(getLogger(), getLogPrefix() + Optional.ofNullable(prefix).map(" "::concat).orElse(""));
+  }
```

Comment:
Is this method shared with any other task (Mojo)?

Reply:
In Maven Plugins, this method is in AbstractJKubeMojo which gets used by Apply, Debug, Log and Migrate goals

Reply:
OK; leave it there and we'll revisit later on.